image.go

```diff
@@ -362,6 +472,87 @@ func (img *imageHDU) Write(data interface{}) error {
 	return err
 }
 
+// ToImage returns a pure Go image.Image instance from the FITS image.
+func (img *imageHDU) ToImage() image.Image {
+
+	var returnedImage image.Image
```

Comment:
it is customary to have shorter variable names in `Go` (see: http://talks.golang.org/2014/names.slide#1)

I would just call it `img`.